src/pages/experiments/[experimentId]/plots-and-tables/components/LegendEditor.jsx

```diff
@@ -1,71 +1,79 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Radio, Form } from 'antd';
+import _ from 'lodash';
+
+const defaultOption = {
+  positions: 'corners',
+  showEnableOption: true,
+  showValueOption: true,
```

Comment:
Agi to double-check on this

Reply:
The 2 values are no longer required, so they're removed.